src/components/ens-profile/ActionButtons/MoreButton.tsx

```diff
@@ -139,7 +139,7 @@ export default function MoreButton({
       if (actionKey === ACTIONS.REMOVE_CONTACT) {
         showDeleteContactActionSheet({
           address,
-          nickname: contact.nickname,
+          nickname: contact!.nickname,
```

Comment:
Similar problem. Contact is defined as:

```ts
const contact = address ? contacts[address.toLowerCase()] : undefined;
```
So if address was undefined then `contact` is undefined and promising TS that `contact` exists can lead to errors. I know it probably won't, due to the way we specify the menu options, but again, if something changes then we can easily create bugs here.